README.md

```diff
@@ -1,15 +1,12 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
```

Comment:
[OPTIONAL] Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), as asked in the requirement : 
<img width="487" alt="image" src="https://user-images.githubusercontent.com/7243132/170134480-19d0832f-be49-4f82-89e9-46475f25e5b8.png">

Here is an example of a professional short summary of changes introduced in the pull request.
<img width="750" alt="image" src="https://user-images.githubusercontent.com/7243132/170134728-ffc1c1ab-5760-443d-971f-a732bfbaafe7.png">

You can use it to improve your summary.
<img width="604" alt="image" src="https://user-images.githubusercontent.com/7243132/170134989-a7d0d661-db84-49aa-ba06-d00aad6b0159.png">